client/src/router/index.ts

```diff
@@ -1,16 +1,20 @@
 import { useRouteMatch } from 'react-router-dom';
 import HomeView from 'views/HomeView';
+import { RouteInfo } from './route';
 import { routes as V2Routes } from './v2';
 
 /**
- * @typedef {object} VersionInfo
  * @property {string} name Bike version friendly name
  * @property {string} rootPath Base path for the version
  * @property {import('./route').RouteInfo[]} routes List of routes under this version
```

Comment:
As before, move these docs inline (and don't need the types)

Reply:
Done
017e912